tests/test_tnsim.py

```diff
@@ -302,6 +302,27 @@ def test_cnot(self):
         value2 = tn_mbqc.expectation_value(random_op2, [0, 1])
         np.testing.assert_almost_equal(value1, value2)
 
+    def test_ccx(self):
```

Comment:
I think it's best to test for reasonably randomized states (c.f. see earlier discussions of this PR and related PRs). Can you add randomizing circuit before the circuit here?